contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Getting started with web analytics
+sidebar: Docs
+showTitle: true
+---
+
+It's easy to get up and running with [web analytics](/web-analytics). All you need to do is [install PostHog](/docs/getting-started/install) and you'll automatically start collecting data based on traffic to your app or website.
```

Comment:
I dont think we need this, given that the previous page is "installation", so lets assume they've already installed posthog

Reply:
(ignore this comment if you rewrite as a tutorial)